templates/metastore.yml

```diff
@@ -0,0 +1,102 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Metastore Stack
```

Comment:
This cluster runs Hive (hiveserver2 and metastore) which lets us just give Presto an endpoint to talk to without having to deal with credentials. It also simplifies running Presto if we move to "not EMR" since we don't have to run a full Hadoop cluster.